keyboards/the_uni/usb_c/readme.md

```diff
@@ -0,0 +1,13 @@
+# The Uni v3 Firmware
+
+-   Keyboard Maintainer: [Peter](https://github.com/petercpark)
+-   Link to Keyboard Files: Not Available
+-   Hardware Supported:
+    -   The Uni v3 (usb_c)
+-   Hardware Availability: [https://www.stenokeyboards.com](https://www.stenokeyboards.com)
+
+Make examples for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```